### PR TITLE
Allow using custom initializers for batch norm params

### PR DESCRIPTION
**Pull request by nanddalal**

### CODE REVIEW

**Comment by vrv** [comment]
closing due to lack of CLA, comment if you can sign

**Comment by nanddalal** [comment]
I have signed now.

**Comment by nanddalal** [comment]
Hi @vrv , just checking in to see if we can merge this?

**Comment by vrv** [comment]
Hey, yeah I now see Clarifai in the corp CLA --is your email part of the google group that is associated with the CLA?   I think that's what can trigger the googlebot to flip the bit

**Comment by vrv** [comment]
Adding Sergio to look at the change to see if it's something we'd be able to accept anyway

File: tensorflow/contrib/layers/python/layers/layers.py [hunk 1]
```diff
@@ -112,6 +112,10 @@ def batch_norm(inputs,
                scale=False,
                epsilon=0.001,
                activation_fn=None,
+               beta_initializer=init_ops.zeros_initializer,
```
Comment (sguada) [inline tensorflow/contrib/layers/python/layers/layers.py]: can you replace this with a dict of initializers which would default to their corresponding value if they key is not present, similar to variables_collections.

**Comment by sguada** [comment]
Thanks, could you add a test where you initialize moving_mean and moving_variance with other values?

**Comment by sguada** [comment]
@tensorflow-jenkins test this please

**Comment by nanddalal** [comment]
@tensorflow-jenkins test this please

**Comment by danmane** [comment]
@tensorflow-jenkins Test this please!

**Comment by vrv** [comment]
I am waiting for the added test before testing, otherwise we'll just have to test again.

**Comment by martinwicke** [comment]
@nanddalal Can you add the test that @sguada requested?

**Comment by nanddalal** [comment]
Hi, I've added a test case for the custom initializers.

**Comment by martinwicke** [comment]
Thanks! @sguada can you review again?

**Review by sguada — APPROVED** [review]
Looks good, thanks.

**Comment by sguada** [comment]
@tensorflow-jenkins Test this please!
